### PR TITLE
[regression test](broker laod) add exception case for  merge type

### PR DESCRIPTION
**Pull request by liugddx**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by liugddx** [comment]
@dataroaring PTAL

File: regression-test/suites/load_p0/broker_load/test_broker_load_with_merge.groovy [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+        // case4: agg table and merge type
+        try {
+            sql "DROP TABLE IF EXISTS ${testTable}"
+            create_agg_test_table.call(testTable)
+
+            def test_load_label = UUID.randomUUID().toString().replaceAll("-", "")
+
+            try {
+                load_from_hdfs_check_merge_type_4.call(testTable, test_load_label, hdfs_csv_file_path, "csv",
```
Comment (morningman) [inline regression-test/suites/load_p0/broker_load/test_broker_load_with_merge.groovy:247]: if `load_from_hdfs_check_merge_type_4` does not throw exception,
your `assertTrue` will not be executed

**Comment by liugddx** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by freemandealer — APPROVED** [review]
LGTM

**Pull request merged**